### PR TITLE
Added misc variables for GUI

### PR DESCRIPTION
**Pull request by zfih**
Adds 4 server related variables as well as health for GUI, Discord RPC, etc.

### CODE REVIEW

**Comment by zfih** [comment]
Unsure if the defaults for the server related variables are sensible.

File: src/main/kotlin/me/zeroeightsix/kami/gui/text/VarMap.kt [hunk 1]
```diff
@@ -80,6 +80,10 @@ object VarMap {
                 mc.networkHandler?.getPlayerListEntry(it.uuid)?.latency?.toDouble()
             } ?: -1.0
         },
+        "serverbrand" string { mc.player?.serverBrand.toString() },
+        "servername" string { mc.currentServerEntry?.name ?: "Singleplayer"},
+        "serverip" string { mc.currentServerEntry?.address ?: "" },
+        "serverversion" string { mc.currentServerEntry?.version?.string ?: mc.gameVersion ?: "" },
```
Comment (5HT2) [inline src/main/kotlin/me/zeroeightsix/kami/gui/text/VarMap.kt]: these should be using an `_` to separate the words, similar to `fps_fast` which is above

**Review by 5HT2 — CHANGES_REQUESTED** [review]

File: src/main/kotlin/me/zeroeightsix/kami/gui/text/VarMap.kt (hunk 1 above)
Comment (zfih) [reply src/main/kotlin/me/zeroeightsix/kami/gui/text/VarMap.kt]: Ah shit yeah that makes sense.

Comment (5HT2) [reply src/main/kotlin/me/zeroeightsix/kami/gui/text/VarMap.kt]: Also the default server ip if null should just be "Offline" or something

**Review by zeroeightysix — APPROVED** [review]

File: src/main/kotlin/me/zeroeightsix/kami/gui/text/VarMap.kt (hunk 1 above)
Comment (zfih) [reply src/main/kotlin/me/zeroeightsix/kami/gui/text/VarMap.kt]: 👍

**Review by 5HT2 — APPROVED** [review]

**Comment by zeroeightysix** [comment]
Thanks!

**Pull request merged**